File: 프로그래머스/unrated/181930. 주사위 게임 2/주사위 게임 2.py
def solution(a, b, c):
    answer = a + b + c
    
    if a==b and b==c:
        answer = answer * a*a*3 * a*a*a*3
    elif a==b or b==c or a==c:
        answer *= (a*a + b*b + c*c)
    
    return answer

File: 프로그래머스/unrated/181930. 주사위 게임 2/주사위 게임 2.java
class Solution {
    public int solution(int a, int b, int c) {
        int answer = 0;
        
        if (a != b && b != c && a != c) {
            answer = a + b + c;
        } else if (a == b && b == c) {
            answer = a*3 * a*a*3 * a*a*a*3;
        } else if (a == b || b == c || a == c) {
            answer = (a + b + c) * (a*a + b*b + c*c);
        }
        
        return answer;
    }
}